### PR TITLE
GOB: Detection improvements

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

File: engines/gob/detection/tables_littlered.h [hunk 1]
```diff
@@ -252,7 +252,7 @@
 		"",
 		AD_ENTRY2s("intro.stk",		"0b72992f5d8b5e6e0330572a5753ea25", 256490,
 				   "mod.babayaga",	"43484cde74e0860785f8e19f0bc776d1", 60248),
-		UNK_LANG,
+		EN_ANY,
```
Comment (tag2015) [inline engines/gob/detection/tables_littlered.h:255]: The Amiga release is multilanguage (see the [Hall of Light entry](https://amiga.abime.net/games/view/little-red-riding-hood), which is a database of Amiga games), so I don't think it should have the English flag

Comment (BJNFNE) [reply engines/gob/detection/tables_littlered.h:255]: Thank your response, then i change it back to UNK_LNG

Comment (BJNFNE) [reply engines/gob/detection/tables_littlered.h]: fixed

File: engines/gob/gameidtotype.h [hunk 2]
```diff
@@ -65,6 +65,7 @@ static const GameIdToType gameIdToType[] = {
 	{               "playtnck2", kGameTypePlaytoons },
 	{               "playtnck3", kGameTypePlaytoons },
 	{           "playtoonsdemo", kGameTypePlaytoons },
+	{           "pierresmagiques", kGameTypeNone },
```
Comment (bluegr) [inline engines/gob/gameidtotype.h:68]:
```suggestion
	{         "pierresmagiques", kGameTypeNone },
```

File: engines/gob/gameidtotype.h [hunk 3]
```diff
@@ -76,6 +77,7 @@ static const GameIdToType gameIdToType[] = {
 	{                 "adibou3", kGameTypeNone },
 	{           "adiboucuisine", kGameTypeNone },
 	{            "adiboudessin", kGameTypeNone },
+	{            "adiboumagie", kGameTypeNone },
```
Comment (bluegr) [inline engines/gob/gameidtotype.h:80]:
```suggestion
	{             "adiboumagie", kGameTypeNone },
```

**Comment by bluegr** [comment]
Thanks!

**Pull request merged**